ui/apps/platform/src/hooks/useCentralCapabilities.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { selectors } from 'reducers';
+import { CentralCapabilitiesFlags } from 'services/MetadataService';
+
+type UseCentralCapabilityResult = {
+    isCentralCapabilityAvailable: (centralCapabilityFlag: CentralCapabilitiesFlags) => boolean;
+};
+
+function useCentralCapabilities(): UseCentralCapabilityResult {
+    const centralCapabilities = useSelector(selectors.getCentralCapabilities);
+
+    const isCentralCapabilityAvailable = useMemo(
+        () =>
+            (centralCapabilityFlag: CentralCapabilitiesFlags): boolean => {
+                const centralCapacity = centralCapabilities[centralCapabilityFlag];
+                if (centralCapacity === 'CapabilityAvailable') {
+                    return true;
+                }
+
+                return false;
```

Comment:
In a Slack discussion, Misha clarified that all features are "on" by default, and a feature has to be explicitly disabled.

I think it would be safer to mimic that logic on the front end, that is, reverse the default return to `true` and only return `false` if 'CapabilityDisabled'.

> The capabilities are provided as Available by default. After you set env variable ROX_MANAGED_CENTRAL to true on Central deployment/pod, the values should be served as Disabled.